.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,127 @@
+name: Create Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      source_ref:
+        description: 'Source branch, commit, or tag to create the release branch from'
+        default: 'develop'
+        type: string
+      release_version:
+        description: 'In the vX.X.X format and will be used for tag and branch name'
+        default: 'vx.x.x'
```

Comment:
non-blocking: Is there any means of validating the presence of the `v` prefix? I'm anticipating `python3 scripts/bump_version.py ${RELEASE_VERSION:1}` doing something bad if it gets forgotten.

Reply:
Yes, I set this input as a variable in line 17: 
```
env:
  RELEASE_VERSION: ${{ inputs.release_version }}
```
and it gets checked when it's first used in the second step:
```

      - name: Create Release Branch
        id: create-release-branch
        run: |
          if [[ $RELEASE_VERSION =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```
If it doesn't match the `^v` then it will fail the build after checking out the code.

Reply:
Awesome, sorry I missed that!